src/dft/src/replace/ScanReplace.cpp

```diff
@@ -360,15 +362,28 @@ void ScanReplace::scanReplace(odb::dbBlock* block)
       continue;
     }
 
-    if (!utils::IsSequentialCell(db_network_, inst)) {
-      // If the cell is not sequential, then there is nothing to replace
-      continue;
-    }
-
     odb::dbMaster* master = inst->getMaster();
     sta::Cell* master_cell = db_network_->dbToSta(master);
     sta::LibertyCell* from_liberty_cell = db_network_->libertyCell(master_cell);
 
+    if (from_liberty_cell == nullptr) {
+      // cell doesn't exist in lib, no timing info
+      if (no_lib_warned.find(master) == no_lib_warned.end()) {
```

Comment:
You could filter useless warning by `master->getType() == dbMasterType::CORE` as no other types make sense for DFT swap.